optimum/exporters/onnx/convert.py

```diff
@@ -77,6 +77,7 @@ def validate_models_outputs(
         Dict[str, Tuple[Union["PreTrainedModel", "TFPreTrainedModel"], OnnxConfig]],
     ],
     output_names: Optional[List[str]] = None,
+    input_shapes: Dict = {},
```

Comment:
Nit: not that it would be an issue here, but dictionaries as default argument values can have weird behaviors https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

Reply:
Oh thanks for the hint